cmd/mockserver/server.go

```diff
@@ -58,31 +71,44 @@ func server(port string) error {
 			continue
 		}
 		log.Println("Accepted connection from", conn.RemoteAddr())
+		// Run this in a goroutine to accept multiple connections at once
```

Comment:
Nit: maybe add TODO to the comment, or just remove it. Otherwise it seems like there's a mismatch between comment and code

Reply:
I am just going to do this now. I think our design should work as is. The test already launches two clients and passes if I just add the "go" here.